.github/workflows/build.yml

```diff
@@ -12,14 +12,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Install node 20
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: 20
+
+      - name: Enable and use Corepack for Yarn
+        run: |
+          corepack enable
+          corepack prepare yarn@4.8.1 --activate
```

Comment:
I wonder if placing the yarn version in package.json might be nice, could be picked-up in local dev.

Reply:
yep, set up the `packageManager` field which should take care of it.